dev_guide/builds/build_inputs.adoc

```diff
@@ -333,14 +333,23 @@ A URI pattern must consist of:
 
 In all of the above, a `*` character is interpreted as a wildcard.
 
-[NOTE]
+[IMPORTANT]
 ====
 URI patterns only match Git source URIs which are conformant to RFC3986. For example,
-https://github.com/openshift/origin.git. They do not match the alternate SSH
-style that Git also uses. For example, git@github.com:openshift/origin.git.
+`https://github.com/openshift/origin.git`. They do not match the alternate SSH
+style that Git also uses. For example, `git@github.com:openshift/origin.git`.
 
 It is not valid to attempt to express a URI pattern in the alternate style, or
 to include a username/password component in a URI pattern.
```

Comment:
The above is expletive. Maybe something like "Expressing a URI pattern in an alternate style, or including a username/password component in a URI pattern will result in an invalid pattern".